.github/workflows/dev-checks.yml

```diff
@@ -581,47 +588,52 @@ jobs:
                   const path = `./${directories[i]}/convoy.yaml`
                   const contentDir = fs.readFileSync(path, 'utf8');
                   const dataDir = yaml.load(contentDir);
-                  exporter = dataDir.artifactNames
+                  exporter = dataDir.generates.artifacts[0].name
                   version = dataDir.version
                   try{
-                    checkVersion('tessell-repos-m2-component',exporter,version);
+                    checkVersion('${{vars.NEXUS_PUSH_REPOS_M2}}',exporter,version);
                   } catch (e) {
                     console.error(e);
                     process.exit(1);
                   }
                 }
                 process.exit(0);
               }
-              else if(artifactType === 'helmChart'){
-                const path = data.chartPath
-                const contentHelm = fs.readFileSync(path, 'utf8');
-                const dataHelm = yaml.load(contentHelm);
-                version = dataHelm.version
+              else if(language === 'helm'){
+                const buildFunction = data.generates.helmCharts[0].buildFunction
+                if (buildFunction === 'helm-template' && '${{steps.changed-files.outputs.any_modified}}' === 'false'){
```

Comment:
Where is `any_modified` output defined? 

In `convoy-helm-template` repo, there is no `buildFunction` key in `convoy.yaml`. Is that an error to fix in that repo?

Reply:
`any_modified` is inbuilt provided by the action used to detect changed files no explicit need to define. 
For convoy-helm-template, not an error it's not defined knowingly since it doesn't builds anything to publish and on the contrary generates key is there because of need of chart path. Me and @SushobhanTessell going to discuss that later to find more logical way for incorporating it.